src/test/app/PayStrand_test.cpp

```diff
@@ -847,24 +847,23 @@ struct PayStrandAllPairs_test : public beast::unit_test::suite
     void
     run() override
     {
-        testAllPairs({featureFlow, fix1373});
-        testAllPairs({featureFlow, fix1373, featureFlowCross});
+        testAllPairs(jtx::supported_features_except (featureFlowCross));
+        testAllPairs(jtx::supported_amendments());
     }
 };
 
 BEAST_DEFINE_TESTSUITE_MANUAL(PayStrandAllPairs, app, ripple);
 
 struct PayStrand_test : public beast::unit_test::suite
 {
-    static bool hasFeature(uint256 const& feat, std::initializer_list<uint256> args)
+    static bool hasFeature(uint256 const& feat, FeatureBitset features)
     {
-        for(auto const& f : args)
-            if (f == feat)
-                return true;
-        return false;
+        FeatureBitset const featBit = jtx::with_only_features (feat);
+        return (features & featBit) == featBit;
```

Comment:
This can be simplified to:`return features[featureToBitsetIndex(feat)];`. There are also three of the `hasFeature` functions. Let's get rid of the three and just put one inside `Feature.h` and `Feature.cpp`